go.mod

```diff
@@ -35,6 +35,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.17.0
 )
 
+require github.com/evanphx/json-patch v4.12.0+incompatible // indirect
```

Comment:
Did you add this directly into the go.mod yourself? Curious why it's in a different block than the rest

Reply:
go mod tidy added it. I imported sigs.k8s.io/controller-runtime/pkg/client/fake and its used there.

Reply:
Can we just manually shift it down? I think if you shift it into the lower block, `go mod tidy` should just respect where you put it IIRC.